.github/workflows/lint.yaml

```diff
@@ -0,0 +1,43 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.3.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --target-branch ${{ github.event.repository.default_branch }} --validate-maintainers=false
+
+#      - name: Create kind cluster
+#        uses: helm/kind-action@v1.4.0
+#        if: steps.list-changed.outputs.changed == 'true'
+#
+#      - name: Run chart-testing (install)
+#        run: ct install --all
```

Comment:
can this get removed?

Reply:
This can be enabled as soon as liveness / readyness checks work as expected and `ct install` successfully ran locally. Therefore I propose to leave this as a reminder.